### ISSUE TITLE
Date picker keyboard navigation

### ISSUE THREAD

**Issue by LiamGallagher737** [thedevdojo/pines#34]
The [date picker](https://devdojo.com/pines/docs/date-picker) component does not have any keyboard navigation currently or anyway to input other that using a pointer device. Could keyboard navigation please be added?

**Comment by kdiffin** [comment]
+1 on this
